sixtyfps_compiler/build.rs

```diff
@@ -36,7 +47,13 @@ fn widget_library() -> &'static [(&'static str, &'static BuiltinDirectory<'stati
         )?;
     }
 
-    writeln!(file, "]\n}}")?;
+    writeln!(file, "]\n}}\n\n")?;
+
+    writeln!(
+        file,
+        "pub(crate) fn builtin_styles() -> &'static [&'static str] {{ &[\"{}\"] }}\n",
```

Comment:
Why do we need this function at all when the information is already in the `widget_library` function

Reply:
Implemented.